Sun Dec  2 14:27:00 FET 2012  Sergei Trofimovich <slyfox@community.haskell.org>
  * fix --flag=testprog depends
  
  Fixes the following failure:
  
  $ runhaskell Setup.hs configure --flags=testprog && runhaskell Setup.hs build
  Preprocessing executable 'shake' for shake-0.3.6...
  
  Development/Shake/FileTime.hs:46:8:
      Could not find module `System.Posix.Files.ByteString'
      It is a member of the hidden package `unix-2.6.0.0'.
      Perhaps you need to add `unix' to the build-depends in your .cabal file.
      Use -v to see a list of the files searched for.
diff -rN -u old-shake/shake.cabal new-shake/shake.cabal
--- old-shake/shake.cabal	2012-12-02 14:29:37.893130874 +0300
+++ new-shake/shake.cabal	2012-12-02 14:29:37.914130875 +0300
@@ -128,6 +128,12 @@
         transformers >= 0.2 && < 0.4,
         deepseq >= 1.1 && < 1.4
 
+    if flag(portable)
+        cpp-options: -DPORTABLE
+    else
+        if !os(windows)
+            build-depends: unix >= 2.5.1
+
     other-modules:
         Examples.Util
         Examples.C.Main
